src/test/scala/io/iohk/ethereum/vm/OpCodeGasSpec.scala

```diff
@@ -437,5 +440,45 @@ class OpCodeGasSpec extends FunSuite with OpCodeTesting with Matchers with Prope
     }
   }
 
+  test(SELFDESTRUCT) { op =>
+    val stateGen = getProgramStateGen(
+      stackGen = getStackGen(elems = 2)
+    )
+
+    // Sending refund to an account created in current transaction
```

Comment:
Good test but the description is inaccurate. `in current transaction` could mean on message call prior to this self destruct operation, in which case we don't charge gas for new account. Just say `// Sending refund to a non-existent account`.